packages/language-server/src/server.ts

```diff
@@ -50,12 +52,14 @@ export function startServer(options?: LSOptions) {
   const documents: TextDocuments<TextDocument> = new TextDocuments(TextDocument)
 
   // Does the clients accepts diagnostics with related information?
-  let hasCodeActionLiteralsCapability: boolean = false
+  let hasCodeActionLiteralsCapability = false
```

Comment:
why change to untyped? (or does initializing with `false` take care of that)

Reply:
that's also unrelated stuff I'll remove, but yes, initializing with false takes care of that.